test/unit/active_model_serializers/register_jsonapi_renderer_test_isolated.rb

```diff
@@ -38,7 +38,7 @@ class WithoutRenderer < JsonApiRendererTest
       require 'active_record'
       require 'support/rails5_shims'
       require 'active_model_serializers'
-      require 'fixtures/poro'
+      require_relative '../../support/fixtures/poro'
```

Comment:
cool, hopefully that fixes it. I don't know why I want straigh to ../support. :-\ (I had to check the commit history to make sure I didn't already try `require 'support/....`